app/javascript/stylesheets/application.scss

```
@@ -314,32 +360,39 @@
     }
+
     &__searchWrap {
       @apply flex items-center;
     }
+
     &__searchInnerWrapper {
       @apply relative mt-1 rounded-md shadow-sm md:ml-5;
+
       input {
         @apply bg-muted text-tiny font-medium;
         width: 224px;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)